src/EditorFeatures/CSharpTest/CommentSelection/CSharpToggleLineCommentCommandHandlerTests.cs

```diff
@@ -235,7 +235,7 @@ class C
 {
     void M()
     {
-[|        //var i = 1; // A comment.|]
+        [|//var i = 1; // A comment.|]
```

Comment:
This has moved the caret and changed the selection. Please leave the selection unchanged. Caret should remain hard left against "var", which is where it was before applying the action.